ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-pim-sparse-mode.j2

```diff
@@ -22,4 +22,16 @@ router pim sparse-mode
       ssm range {{ router_pim_sparse_mode.ipv4.ssm_range }}
 {%         endif %}
 {%     endif %}
+{%     for vrf in router_pim_sparse_mode.vrfs | arista.avd.natural_sort('name') %}
+      !
+    vrf {{ vrf.name }}
+{%         if vrf.ipv4 is arista.avd.defined %}
+      ipv4
+{%             for rp_address in vrf.ipv4.rp_addresses | arista.avd.natural_sort('address') %}
+{%                 for group in rp_address.groups | arista.avd.default([]) | arista.avd.natural_sort %}
```

Comment:
When leveraging ```natural_sort``` filter we don't to check if the knob is defined or not because the natural_sort filter itself does that check. https://avd.sh/en/releases-v3.x.x/plugins/index.html#natural_sort-filter

So please change this line 31 to: ```for group in rp_address.groups | arista.avd.natural_sort```